lib/first-mate.js

```diff
@@ -1,14 +1,12 @@
-(function() {
-  module.exports = {
-    ScopeSelector: require('./scope-selector'),
-    GrammarRegistry: require('./grammar-registry'),
-    Grammar: require('./grammar')
-  };
 
-  Object.defineProperty(module.exports, 'OnigRegExp', {
-    get: function() {
-      return require('oniguruma').OnigRegExp;
-    }
-  });
+module.exports = {
+  ScopeSelector: require("./scope-selector.js"),
+  GrammarRegistry: require("./grammar-registry.js"),
+  Grammar: require("./grammar.js")
+};
 
-}).call(this);
+Object.defineProperty(module.exports, 'OnigRegExp', {
+  get: function() {
+    return require('oniguruma').OnigRegExp;
```

Comment:
You can also use a newer style for this:
```js
module.exports = {
  ScopeSelector: require('./scope-selector'),
  GrammarRegistry: require('./grammar-registry'),
  Grammar: require('./grammar'),
  get OnigRegExp(){
    return require('oniguruma').OnigRegExp;
  }
};
```

Reply:
Great catch there, I'll go ahead and add it in. But thanks for the review and if it's looking good I'll keep going in this style and hopefully is anyone else working on decaf work